Documentation/01-restore-etcd-from-backups.md

```diff
@@ -108,6 +108,53 @@ etcdctl --endpoints https://127.0.0.1:2379 member add 00000002 https://172.16.23
 etcdctl --endpoints https://127.0.0.1:2379 member list
 ```
 
+## Restoring a guest cluster (single etcd member)
+
+### Copy db backup from s3
+Find the [etcd backup to restore](https://s3.console.aws.amazon.com/s3/buckets/etcd-backups.giantswarm.io/?region=eu-central-1&tab=overview) and make it public. Copy the link and download in the master (wget). After download it, go to the permission tab and removes `Read` right for everyone to leave as before (not public).
```

Comment:
Feels weird having this in a public repo

Reply:
Also, the backup procedure involves making the backup public? That can't be right

Reply:
well let's omit that part, I did not realize it is public. 

> Also, the backup procedure involves making the backup public? That can't be right

well using the creds is the right approach. But I thought in case of emergency you can make it public for downloading and remove the right after.